Networking/Networking/Remote/CouponsRemote.swift

```diff
@@ -372,6 +372,23 @@ public final class CouponsRemote: Remote, CouponsRemoteProtocol {
     }
 }
 
+extension CouponsRemote {
+    public func loadAllCoupons(for siteID: Int64,
```

Comment:
super nit; not loading all coupons here

```suggestion
    public func loadCoupons(for siteID: Int64,
```

Reply:
I'm not sure to understand: `loadAllCoupons` loads all while `loadCoupons` loads a subset which we know the couponIDs of, right? _"makes a single request for a list of coupons"_ but we don't have that list so we have to fetch them all.

Reply:
I just meant from a readability point of view. `loadAllCoupons` sounds like we're loading ALL coupons, rather than specific page. But I see now that `CouponsRemoteProtocol` has `loadAllCoupons` and `loadCoupons` which is different. 👍 So we can keep it as it is. 👍

Reply:
Ah gotcha, I agree the names are a bit confusing thought, I'll see if I can sneak a task separately to rename these better.

Reply:
It's fine, not necessary 👍